test/integration/targets/junos_l2_interface/tests/netconf/basic.yaml

```diff
@@ -0,0 +1,337 @@
+---
+- debug: msg="START junos_l2_interface netconf/basic.yaml on connection={{ ansible_connection }}"
+
+- name: setup - remove interface address
+  junos_l2_interface:
+    name: ge-0/0/1
+    state: absent
+    provider: "{{ netconf }}"
+
+- name: setup - remove interface address
+  junos_l2_interface:
+    name: ge-0/0/2
+    state: absent
+    provider: "{{ netconf }}"
+
+- name: Setup delete vlan 100
```

Comment:
Why bother deleting them before creating them? Is `junos_vlan` not idempotent, or are there other considerations here?